perftests/https/Makefile.BSD

```diff
@@ -0,0 +1,66 @@
+PROG=	test_https
+SRCS=	main.c
+
+# Useful relative directories
+LIBCPERCIVA_DIR	=	../../libcperciva
+LIB_DIR	=	../../lib
+
+# Library code required
+LDADD_REQ	= -lssl -lcrypto
+
+# Data structures
+.PATH.c	:	${LIBCPERCIVA_DIR}/datastruct
+SRCS	+=	elasticarray.c
+SRCS	+=	ptrheap.c
+SRCS	+=	timerqueue.c
+IDIRS	+=	-I ${LIBCPERCIVA_DIR}/datastruct
+
+# Utility functions
+.PATH.c	:	${LIBCPERCIVA_DIR}/util
+SRCS	+=	asprintf.c
+SRCS	+=	monoclock.c
+SRCS	+=	sock.c
+SRCS	+=	warnp.c
+IDIRS	+=	-I ${LIBCPERCIVA_DIR}/util
+
+# Event loop
+.PATH.c	:	${LIBCPERCIVA_DIR}/events
+SRCS	+=	events_immediate.c
+SRCS	+=	events_network.c
+SRCS	+=	events_network_selectstats.c
+SRCS	+=	events_timer.c
+SRCS	+=	events.c
+IDIRS	+=	-I ${LIBCPERCIVA_DIR}/events
+
+# Event-driven networking
+.PATH.c	:	${LIBCPERCIVA_DIR}/network
+SRCS	+=	network_connect.c
+SRCS	+=	network_read.c
+SRCS	+=	network_write.c
+IDIRS	+=	-I ${LIBCPERCIVA_DIR}/network
+
+# SSL layer
+.PATH.c	:	${LIB_DIR}/network_ssl
+SRCS	+=	network_ssl.c
+SRCS	+=	network_ssl_compat.c
+IDIRS	+=	-I ${LIB_DIR}/network_ssl
+
+# Buffered networking
+.PATH.c	:	${LIB_DIR}/netbuf
+SRCS	+=	netbuf_read.c
+SRCS	+=	netbuf_write.c
+SRCS	+=	netbuf_ssl.c
+IDIRS	+=	-I ${LIB_DIR}/netbuf
+
+# HTTP protocol
+.PATH	:	${LIB_DIR}/http
+SRCS	+=	http.c
+SRCS	+=	https.c
+IDIRS	+=	-I ${LIB_DIR}/http
+
+CFLAGS	+=	-g
```

Comment:
Not sure why this is here.  Probably I added it when I was debugging code and forgot to remove it later...?